src/error.rs

```diff
@@ -2,6 +2,7 @@ use thiserror::Error;
 
 /// Crate-specific error enum.
 #[derive(Debug, Error)]
+#[allow(clippy::large_enum_variant)]
```

Comment:
The size difference isn't that big, and for a non-performance-critical repo like this one I think it's ok to let this one slide.

```
  --> src/error.rs:5:1
   |
5  | / pub enum Error {
6  | |     /// [geojson::Error]
7  | |     #[error(transparent)]
8  | |     GeoJSON(#[from] geojson::Error),
   | |     ------------------------------- the largest variant contains at least 320 bytes
...  |
66 | |     Validation(serde_json::Value),
   | |     ----------------------------- the second-largest variant contains at least 72 bytes
67 | | }
   | |_^ the entire enum is at least 320 bytes
   |
```